pegasus/sites.v3/code.org/views/swiper_page_diversity.haml

```diff
@@ -0,0 +1,32 @@
+%script{src: webpack_asset_path('js/code.org/views/swiper_page_diversity.js')}
+
+:ruby
+  videos = [
+    {
+      youtube_code: 'mFPg96gdPkc',
+      title: "Change the World - Hour of Code",
```

Comment:
Because this page isn't translated, I kept these titles in English. Happy to use or create strings for these if folks prefer, though!